src/main/webapp/result.jsp

```diff
@@ -0,0 +1,39 @@
+<%@page import="java.util.*"%>
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<html>
+<head>
+<title>Folders with images</title>
+</head>
+<body>
+	<h3>Previously added folders:</h3>
+	<%
+		ArrayList list = (ArrayList) request.getServletContext().getAttribute("storage");
```

Comment:
There are not java code in JSP, use custom tags instead (http://www.journaldev.com/2099/jsp-custom-tags-example-tutorial)